src/components/Preferences/ModalActions.vue

```diff
@@ -47,4 +89,16 @@ export default Vue.extend({
     padding: var(--preferences-content-padding);
     border-top: 1px solid var(--header-border);
   }
+
+  .banner {
+    margin: 0;
+  }
+
+  .fade-enter, .fade-leave-to {
```

Comment:
Please doc that these classes come from using the name `fade` in `<transition name="fade"...>` , because these class names are synthesized from our code.

Reply:
These classes are provided by Vue for use with the `transition` component https://v2.vuejs.org/v2/guide/transitions.html.

I could add a comment here, but I'm inclined to push back on it a bit because it feels it bit odd to document framework features.